src/pages/api/users/[id].js

```diff
@@ -11,20 +12,29 @@ handler.use(forbiddenUnlessRoot);
 handler.patch(async (req, res) => {
   const {
     query: { id },
-    body: { accessLevel },
+    body: { accessLevel, instructor: instructorId },
   } = req;
 
-  if (!["instructor", "admin", "revoked"].includes(accessLevel)) {
+  if (
+    accessLevel &&
+    !["instructor", "admin", "revoked"].includes(accessLevel)
```

Comment:
If I'm reading it right, I feel like this server side check should have caught the issue I described above :thinking: ?